terragrunt/aws/api/route53.tf

```diff
@@ -37,7 +37,8 @@ module "resolver_dns" {
     "*.amazonaws.com.",
     "*.cyber.gc.ca.",
     "current.cvd.clamav.net.",
```

Comment:
Won't we need the cloudfare dns for this guy too?

Reply:
I checked and that one resolves to an `A` record directly so we're good:
```sh
❯ dig current.cvd.clamav.net

;; QUESTION SECTION:
;current.cvd.clamav.net.		IN	A

;; AUTHORITY SECTION:
cvd.clamav.net.		1800	IN	SOA	ns3.clamav.net. hostmaster.oltrelinux.com. 2006452102 1800 900 604800 7200
```